tools/slo-generator/slo_generator/compute.py

```diff
@@ -192,20 +203,23 @@ def make_measurement(slo_config, step, good_event_count, bad_event_count,
     timestamp_human = utils.get_human_time(timestamp)
 
     # Compute SLI and gap between SLI / SLO target.
-    sli = good_event_count / (good_event_count + bad_event_count)
     gap = sli - slo_target
 
     # Compute Error Budget (target, current value, remaining minutes, available
     # minutes).
     error_budget_target = 1 - slo_target
     error_budget_target = 1 - slo_target
-    error_budget_measurement = 1 - sli
+    error_budget_value = 1 - sli
     error_budget_remaining_minutes = window * gap / 60
-    error_minutes = window * error_budget_measurement / 60
+    error_minutes = window * error_budget_value / 60
     error_budget_minutes = window * error_budget_target / 60
 
     # Compute Error Budget Burn rate: the % of consumed error budget.
-    error_budget_burn_rate = error_budget_measurement / error_budget_target
+    if error_budget_target == 0:
+        error_budget_burn_rate = 0
+    else:
+        error_budget_burn_rate = round(error_budget_value / error_budget_target,
+                                       1)
```

Comment:
This formatting looks odd, I'd suggest splitting separately if you need two lines.

Reply:
YAPF did this automatically, if I change it it complains that the code is not formatted properly...

Reply:
Does this work?
```
error_budget_burn_rate = round(
  error_budget_value / error_budget_target, 1)
```

Reply:
Works ! Committed.

Reply:
Nevermind, when running `make` on the repo root it restores it to the previous ...